rules/windows/file/file_event/file_event_win_creation_system_file.yml

```diff
@@ -89,48 +90,38 @@ detection:
             - '\WSReset.exe'
             - '\WUDFHost.exe'
             - '\WWAHost.exe'
-    filter_main_dism_tiworker:
+    filter_main_generic:
+        # Note: It is recommended to use a more robust filter instead of this generic one, to avoid false negatives.
         TargetFilename|contains:
-            - ':\Windows\SoftwareDistribution\'
-            - ':\Windows\System32\'
-            - ':\Windows\SysWOW64\'
-            - ':\Windows\WinSxS\'
             - '\SystemRoot\System32\'
-        Image|endswith:
-            - '\Windows\System32\dism.exe'
-            - '\TiWorker.exe'
-    filter_main_setuphost:
-        TargetFilename|contains: ':\$WINDOWS.~BT\'
-        Image|endswith: ':\$WINDOWS.~BT\Sources\SetupHost.exe'
-    filter_main_wbengine:
-        TargetFilename|endswith: '\RuntimeBroker.exe'
-        Image|endswith: ':\Windows\system32\wbengine.exe'
+            - 'C:\$WINDOWS.~BT\'
+            - 'C:\$WinREAgent\'
+            - 'C:\Windows\SoftwareDistribution\'
+            - 'C:\Windows\System32\'
+            - 'C:\Windows\SysWOW64\'
+            - 'C:\Windows\WinSxS\'
+            - 'C:\Windows\uus\'
```

Comment:
In filters we decide to keep it. To avoid false negatives that arise from multi disk systems. An attacker can create a similar structure on `C:\Windows\uus\` for example.